.travis.yml

```diff
@@ -29,6 +29,7 @@ jobs:
           python-dateutil==2.8.0
           wcwidth==0.1.7
           psutil==5.4.3
+          semantic_version==2.8.5
```

Comment:
Go with version = 2.6.0, that's the default on my system. We'll bump them all when we move up to Fedora 31.

Reply:
Agreed; 2.6.0 will be what's installed on the test systems now.

(I'm re-running the test after installing it.)

Reply:
I'm getting this failure:
```
ImportError: cannot import name 'SimpleSpec' from 'semantic_version' (/usr/lib/python3.7/site-packages/semantic_version/__init__.py)
```

Reply:
Ok. Good catch. We will have to be 2.6.0 compatible, regardless of our recommended development version. It looks like there's some prior version alternatives to SimpleSpec.

Reply:
I believe SimpleSpec was introduced in version 2.7.0

Reply:
Replacing SimpleSpec with Spec should fix this. Testing now